docs/reference/items.md

```diff
@@ -164,7 +164,7 @@ text: |
 When this item is published, Doorstop will create a new heading with the text
 "1.1.0 This is the heading" and put the remaining text into its body.
 
-#### Example: Normative item
+#### Example: Normative heading
```

Comment:
I believe this will only be rendered as a heading if there are items nested under it, correct?

Reply:
```shell
doorstop create SRS srs
doorstop add SRS
doorstop publish SRS one_item.md
doorstop add SRS
doorstop add SRS
doorstop publish SRS three_items.md
```

Result in *one_item.md*:

```
# 1.0 SRS001 {#SRS001 }
```

Result in *three_items.md*:

```
# 1.0 SRS001 {#SRS001 }

## 1.1 SRS002 {#SRS002 }

## 1.2 SRS003 {#SRS003 }

```

It seems it is always rendered as a heading. Or am I missing something?

Reply:
Well, they're all technically "headings" in Markdown. 😄 

The more that I think about this, I think that "Normative heading" will cause more confusion. I'm inclined to close #304 as "working as intended". I would appreciate a cleanup PR that fixes the triple backticks.